spec/models/book_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
```

Comment:
Do you really need this?

Reply:
i do, without this line it doesnt work :/